packages/web/src/components/gcds-stepper/gcds-stepper.tsx

```diff
@@ -17,12 +17,33 @@ export class GcdsStepper {
   /**
    * Defines the current step.
    */
-  @Prop() currentStep!: number;
+  @Prop({ mutable: true }) currentStep!: number;
+  @Watch('currentStep')
+  validateCurrentStep() {
+    if (this.currentStep == undefined) {
+      this.currentStep = 1;
+    } else if (this.currentStep > this.totalSteps) {
+      this.currentStep = this.totalSteps;
+    }
+  }
 
   /**
    * Defines the total amount of steps.
    */
-  @Prop() totalSteps!: number;
+  @Prop({ mutable: true }) totalSteps!: number;
+  @Watch('totalSteps')
+  validateTotalSteps() {
+    if (this.totalSteps == undefined) {
+      this.totalSteps = 3;
+    } else if (this.totalSteps < this.currentStep) {
+      this.totalSteps = this.currentStep;
+    }
```

Comment:
Same comment here as above on validation; I feel like we shouldn't be setting arbitrary values for these because they are a required field, is there a really good reason that we have to set it instead of reporting it as an invalid parameter?